mnist/training/train.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kubeflow.org/v1
+kind: TFJob
+metadata:
+  name: $(trainingName)
+  namespace: kubeflow
```

Comment:
haven't tested yet -- but would this have the same issue of needing to swap in the user profile namespace?

Reply:
Woops this should have been removed. Originally I thought I would have YAML files that I would load in the notebook. But I decided it was more illustrative to inline the YAML in the notebook and use f style strings for substitution. I will remove this.